### DISCUSSION TITLE
can graoh-node store failed transaction？

### DISCUSSION THREAD

**Discussion by baixiaoyu2997** [graphprotocol/graph-tooling#1002] [Q&A]
i can't find any about this , how to store a  faild transaction to subgraph? is this possible？

**Comment by azf20** [comment]
hey @baixiaoyu2997 this isn't currently possible, only successful transactions are indexed. Do you have a use case for failed transactions?

**Reply by baixiaoyu2997** [reply]
thanks for you replay,  just by my product manager， he want  to make a tx list, include success and failed.  he says why `Etherscan` can but we can't ?  lol
